lib/shipwire/orders.rb

```diff
@@ -24,18 +24,10 @@ def holds(id, params = {})
       request(:get, "orders/#{id}/holds", params: params)
     end
 
-    def release(id)
```

Comment:
Also ???

Read the commit message, but didn't follow still

Reply:
ok, so now looking back at this i realize what's going on. I had originally accidentally opened a PR against https://github.com/billr578/shipwire which is where all the differences were coming from. I'll need to revert those commits. thanks for catching this